README.md

```diff
@@ -1,9 +1,23 @@
 # comixcraft - Showcase
 In this repository a showcase of the comixcraft project is saved.
-Currently it is just a placeholder-page. When implementing the placeholder can be replaced by any page, of any chosen technology.
+This website presents the comixcraft editor with its key functionalities and team members. 
+
+## Run the project
+To run the project, double-click on the index.html file to open it in your default web browser. 
+
+## Style Guide
+
+The font used for the **comixcraft**-Showcase is _Nunito_, it is the default for everything.
+Any text in the comics themselves is in the font _Pangolin_.
+
+The colors, spacers, borders and other design elements are defined in our brand guide and only these should be used. They are the same as the ones in the editor. 
```

Comment:
maybe add a link to where to find them in the editor repository

Reply:
You could also add the link to the fonts (https://github.com/comixcraft/Editor/tree/dev/public/fonts)